addons/website/views/snippets/s_popup.xml

```diff
@@ -40,23 +41,26 @@
         <div data-js="SnippetPopup" data-selector=".s_popup" data-target=".s_popup_main">
             <we-select string="Position" class="o_we_inline">
                 <we-button data-set-layout="modal" data-name="position_centered">Center</we-button>
-                <we-button data-set-layout="fixed">Bottom</we-button>
+                <we-button data-set-layout="fixed" data-name="position_bottom">Bottom</we-button>
             </we-select>
             <we-select string="Size" data-apply-to=".s_popup_frame" class="o_we_inline">
                 <we-button data-select-class="modal-sm">Small</we-button>
-                <we-button data-select-class="modal-md">Medium</we-button>
+                <we-button data-select-class="modal-md" data-name="s_popup_size_md">Medium</we-button>
                 <we-button data-select-class="modal-lg">Large</we-button>
                 <we-button data-select-class="modal-xl">Big</we-button>
-                <we-button data-select-class="s_popup_full">Full</we-button>
+                <we-button data-select-class="s_popup_full" data-name="s_popup_size_full">Full</we-button>
             </we-select>
             <we-colorpicker string="Backdrop Color" data-select-style="true"
                 data-css-property="background-color" data-color-prefix="bg-" data-dependencies="position_centered"/>
+        </div>
+        <div data-js="SnippetPopup" data-selector=".s_popup" data-exclude="#website_cookies_bar" data-target=".s_popup_main">
             <we-colorpicker string="Text Color" data-select-style="true" data-css-property="color" data-color-prefix="text-"/>
             <we-select string="Display" data-attribute-name="display" data-attribute-default-value="always" class="o_we_inline">
                 <we-button data-select-data-attribute="afterDelay" data-name="show_delay">Delay</we-button>
                 <we-button data-select-data-attribute="mouseExit">On Exit</we-button>
             </we-select>
             <we-input class="o_we_inline" string="Delay" data-select-data-attribute="" data-attribute-name="showAfter" data-unit="s" data-save-unit="ms" data-dependencies="show_delay"/>
+            <we-input class="o_we_inline" string="Period between consents" data-select-data-attribute="7days" data-attribute-name="consentsDelay" data-unit="days"/>
```

Comment:
Alternatives:
_ Consent Duration
_ Expires after

Reply:
`Consent Duration` IMO

Reply:
As discussed in website office, we will go with `Hide For` for now and a tooltip.
Note that title are broken now, need https://github.com/odoo/odoo/pull/43111 in order to work